User-Help/GlobalTrustLevels.md

```diff
@@ -0,0 +1,138 @@
+<!--- Explanation of what Trust Levels, how they are calculated, and how to earn then -->
+
+# Trust Level Overview
+
+Because Codidact is community run, it doesn't have owners who dictate behavior. Instead, users who consistently make solid contributions through posting and other activity on a community gradually obtain more privileges and access to certain moderation tools based on a calculated "Trust Level." You can see current Trust Level on any user's profile. 
```

Comment:
Technically Codidact has owners who would dictate in extreme cases, like no we won't let you set up a Nazi-lovers community or something like that.

Consider an opening sentence along these lines:

"Codidact sites are run by the community and are built on a system of trust.  Users who consistently... (etc)"